src/components/domain/auth/login/LoginModal.tsx

```diff
@@ -43,34 +44,28 @@ export default function LoginModal() {
   } = formMethods;
 
   const onSubmit = async (data: LoginForm) => {
```

Comment:
`loginMutation.mutate`에서 이미 비동기 작업을 처리하는데 `async`가 필요한가요..? 
임시 코드 삭제하시면서 `async` 제거가 누락된 것 같습니다.

```suggestion
  const onSubmit = (data: LoginForm) => {
```

Reply:
✅ f1aa2b3ea7f059ba8dbab821d3f15ccbe5d57510